packages/core/src/utils.ts

```diff
@@ -975,24 +995,47 @@ export const toOpenZeppelinProxyType = (
 }
 
 export const getOpenZeppelinValidationOpts = (
-  proxyType: ProxyType
+  proxyType: ProxyType,
+  contractConfig: UserContractConfig
 ): Required<ValidationOptions> => {
-  return {
+  type UnsafeAllow = Required<ValidationOptions>['unsafeAllow']
+
+  const unsafeAllow: UnsafeAllow = [
+    'state-variable-assignment',
+    'constructor',
+    'state-variable-immutable',
+    'missing-public-upgradeto',
```

Comment:
> I think we can remove the UUPS `upgradeTo` check [here](https://github.com/chugsplash/chugsplash/blob/3cd22d5cb9da21b0f0247346dde171ffefc91e34/packages/core/src/config/parse.ts#L1256-L1282) since OpenZeppelin does it for us now

If you agree, we should remove `'missing-public-upgradeto`' here

Reply:
See my other comment, will do this in a separate PR